manifests/sar_metric.pp

```diff
@@ -0,0 +1,60 @@
+define puppet_metrics_collector::sar_metric (
+  String                    $output_dir,
+  String                    $scripts_dir,
+  Enum['absent', 'present'] $metric_ensure  = 'present',
```

Comment:
It seems odd to have some of these aligned with extra spaces ($metric_ensure,  $metrics_type, etc...) and the rest not aligned ($polling_frequency_seconds, $collection_frequency). Is this intentional?

Reply:
They look aligned with extra space on my screen?